actionpack/lib/action_dispatch/middleware/server_timing.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "action_dispatch/middleware/server_timing_subscriber"
```

Comment:
I have this `require` here so that when this file is evaluated so is the `server_timing_subscriber`, and so is that classes `attach_to` method. That way a subscriber has always been attached when  we call `ServerTiming`, a similar pattern to the `LogSubscriber`. 

The alternative is to call `ServerTimingSusbcriber.attach_to` directly in the middleware or in this file.